docs/application-developers-reference-guide.md

```diff
@@ -0,0 +1,83 @@
+# Application Developer's Reference Guide
+
+This document is intended for cloud-native application developers targeting application deployments on Bitnami Kubernetes Production Runtime (BKPR). It describes the configuration you need to perform for automating the ingress, TLS, logging and monitoring support for their application.
+
+## Introduction
+
+BKPR is a curated collection of services running on top of your existing Kubernetes cluster with the aim of automating the configuration of public access, logging and monitoring, and management of DNS records and TLS certificates.
+
+For BKPR to perform this automation, you need to apply a few configurations in Kubernetes manifests of your application. The following sections walk you through each of these configurations.
+
+## Ingress and Let's Encrypt
+
+BKPR installs an [NGINX controller](https://github.com/kubernetes/ingress-nginx) which provides load balancing, SSL termination and name-based virtual hosting for your applications. Ingress enables application developers to expose HTTP and HTTPS routes from outside the cluster to services running inside the cluster.
```

Comment:
"Ingress enables you" -> since it looks like the rest of the doc refers to the reader as "you", or adjust as needed